gateway-service-impl/src/main/java/org/hypertrace/gateway/service/common/QueryRequestContext.java

```diff
@@ -32,6 +34,15 @@ public FunctionExpression getFunctionExpressionByAlias(String alias) {
     return aliasToFunctionExpressionMap.get(alias);
   }
 
+  public void mapAliasToMapAttributeExpression(
```

Comment:
this "request context" is a real monster huh

Reply:
oh yes